circle.yml

```diff
@@ -291,11 +295,30 @@ jobs:
       - *update-submodules
       - *environment-info
       - *restore-deps-cache
-      - *configure
```

Comment:
Why remove this?

Reply:
I think adding back this will fix the CI error too.

Reply:
Accident